src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -310,17 +310,32 @@ private static void handleFailedConfigs(
     /**
      * Generates a report at the {@code repoReportDirectory}.
      */
-    public static void generateEmptyRepoReport(String repoReportDirectory, String displayName) {
+    public static List<Path> generateEmptyRepoReport(String repoReportDirectory, String displayName) {
```

Comment:
Same here

Reply:
Done. I have an unrelated question - is there any reason for this method to be public?

Reply:
At this point in time, there's no particular reason. It's probably due to the method being used elsewhere previously that's no longer the case now.

You can change this to private

Reply:
Thanks for the response. It is necessary to create a separate issue for this, since it's technically not related to this PR?

Reply:
@jylee-git, I've created a new issue for this.